gslib/commands/config.py

```diff
@@ -159,35 +159,32 @@
               authentication method before OAuth2 was supported) instead of
               obtaining an OAuth2 token.
 
-  -b          Causes ``gsutil config`` to launch a browser to obtain OAuth2
-              approval and the project ID instead of showing the URL for each
-              and asking the user to open the browser. This will probably not
-              work as expected if you are running gsutil from an ssh window, or
-              using gsutil on Windows.
-
+  -b          DEPRECATED. Formerly caused ``gsutil config`` to launch a browser
```

Comment:
I will wait and see where we end up with the discussion regarding these fields on the CL by Nate.